### PR TITLE
Changelog/version bump for PR #958

### PR DESCRIPTION
**Pull request by Kully**
@cldougl 

NB. version is set to `2.5.0` because the dashboard PR https://github.com/plotly/plotly.py/pull/946 has the same version number

### CODE REVIEW

**Comment by Kully** [comment]
@chriddyp

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.5.0] - UNRELEASED
+### Fixed
+- `import plotly.figure_factory` does not fail if `pandas` is not installed.
```
Comment (chriddyp) [inline CHANGELOG.md]: Could you link to the PR where this was fixed?

Comment (Kully) [reply CHANGELOG.md]: How's `This is fixed in the PR https://github.com/plotly/plotly.py/pull/958`?
or just `https://github.com/plotly/plotly.py/pull/958`

**Comment by cldougl** [comment]
💃

**Pull request merged**